pkg/server/errors/errors.go

```diff
@@ -143,6 +143,11 @@ func HandleTupleValidateError(err error) error {
 		return TypeNotFound(t.TypeName)
 	case *tuple.RelationNotFoundError:
 		return RelationNotFound(t.Relation, t.TypeName, t.TupleKey)
+	case *tuple.InvalidConditionalTupleError:
+		return status.Error(
+			codes.Code(openfgav1.ErrorCode_validation_error),
+			err.Error(),
+		)
```

Comment:
not going to block the PR for this, but if in the future we add a new error to `tuple_errors`, how are we going to remember to update this code here?